packages/apps/graph/src/graph/objects/transaction-result.ts

```diff
@@ -1,6 +1,23 @@
+import { prismaClient } from '@db/prisma-client';
+import { Prisma } from '@prisma/client';
+import {
+  COMPLEXITY,
+  getDefaultConnectionComplexity,
+} from '@services/complexity';
+import { nullishOrEmpty } from '@utils/nullish-or-empty';
 import { builder } from '../builder';
 
-export default builder.objectType('TransactionResult', {
+const MempoolInfo = builder.objectType('MempoolInfo', {
+  description: 'The mempool information.',
+  fields: (t) => ({
+    status: t.exposeString('status', {
+      description: 'The status of the mempool.',
+      nullable: true,
+    }),
+  }),
+});
+
+const TransactionInfo = builder.objectType('TransactionInfo', {
```

Comment:
Is `eventCount` needed here? Since we included the `events` object? 
The same question regarding `blockHash`. 
Here, the problem with removing these is that if we query for these fields via the corresponding objects we will be querying the database an additional time. @alber70g

Reply:
We can remove them from the schema but query them from Prisma right?